.sourcery/LinuxMain.stencil

```diff
@@ -0,0 +1,24 @@
+//
+//  LinuxMain.swift
+//  SwiftLint
+//
+//  Created by JP Simard on 12/11/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
```

Comment:
should we add a note that this file is automatically generated and not modified directly?